Formula/tweak.rb

```diff
@@ -1,18 +1,15 @@
-class Tweak < Formula
+lass Tweak < Formula
   desc "Command-line, ncurses library based hex editor"
   homepage "https://www.chiark.greenend.org.uk/~sgtatham/tweak/"
   url "https://www.chiark.greenend.org.uk/~sgtatham/tweak/tweak-3.02.tar.gz"
   sha256 "5b4c19b1bf8734d1623e723644b8da58150b882efa9f23bbe797c3922f295a1a"
 
   def install
     system "make"
-    bin.install "tweak"
-    man1.install "tweak.1"
-    doc.install "btree.html"
-    prefix.install_metafiles
+    system "make", "install", "PREFIX=#{prefix}", "MANDIR=#{man}"
   end
 
   test do
-    system bin/"tweak", "--help"
+    assert_equal "# Default .tweakrc generated by `tweak -D'.\n", `tweak -D | head -1`
```

Comment:
tried using tweak -D which outputs the default .tweakrc file to stdout ...
alternatively could ... maybe write that to a file? then using tweak to read it? but then couldn't figure out how to compare the contents of an emacs window with asserts

also is it ok to use head?